Makefile

```diff
@@ -10,23 +10,22 @@ install-docker-tools:
 
 install:
 	docker run --rm -v $$(pwd):/app composer install \
-	&& docker-compose exec php yarn install
+	&& docker-compose exec php yarn install \
+	&& docker-compose exec php yarn build \
+	&& docker-compose exec php make dev
```

Comment:
I think that you should not use `docker-compose exec php yarn install` in the `install` but use your `make yarn` and `make yarn-build`

Reply:
Same for `docker-compose exec php make dev` you could directly enter in the container in the `make dev` as in your `yarn-build` and `yarn`

Reply:
Yeah but the server don't have any task runner or package manager installed, all these actions are done on the container

Reply:
Yeah and your `make yarn-build` and `make yarn` are executing commands in your container